README.md

```
@@ -458,166 +508,219 @@ csfd
 
-#### Options for user reviews
+#### UserReviewsOptions
 
-| Option            | Type                                                                                                                                       | Default | Description                                            |
-| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ------- | ------------------------------------------------------ |
-| **includesOnly**  | [CSFDFilmTypes[]](https://github.com/bartholomej/node-csfd-api/blob/8fa5f9cbc7e7f2b62b0bd2c2b5a24c9a63444f6a/src/interfaces/global.ts#L25) | null    | Including only film types. eg. `['seriál', 'koncert']` |
-| **exclude**       | [CSFDFilmTypes[]](https://github.com/bartholomej/node-csfd-api/blob/8fa5f9cbc7e7f2b62b0bd2c2b5a24c9a63444f6a/src/interfaces/global.ts#L25) | null    | Excluding film types eg. `['epizoda', 'série']`        |
-| **allPages**      | boolean                                                                                                                                    | false   | Get all pages                                          |
-| **allPagesDelay** | number                                                                                                                                     | 0       | Delay on each page request. In milliseconds            |
-| **page**          | number                                                                                                                                     | 1       | Specific page number to fetch (e.g., 2 for page 2)     |
+Same options as [UserRatingsOptions](#userrationsoptions).
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)